src/sections/Docker-Meshery/docker-extension-CTA.js

```diff
@@ -7,7 +7,7 @@ import Dockerimage from "../../assets/images/docker-extension/Docker_animated.sv
 
 const DockerExtensionCTAWrapper = styled.div`
 p {
-  color:  black;  
+  color: ${props => props.theme.DarkTheme ? "#fff" : "#000"};   
```

Comment:
@anshitmishra Let's not use conditionals here. Please go through the contributing guide which explains changing colors according to theme. Here's the link: https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-dark-mode-for-components